airbyte-integrations/connectors/source-braintree/source_braintree/schemas/subscription.py

```diff
@@ -15,7 +15,8 @@ class Subscription(CatalogModel):
     add_ons: List[AddOn]
```

Comment:
Pretty wild to see Python schemas — we switched away from them quite some time ago it seems. But hey, if this works, I'm ok. 

I'm worried that the side effect of this could be that we lose some data (the time portion?), but if the API changed and the responses only have dates, that's ok.

Reply:
That first Airbyte issue was posted in June 2022, so if the API did change from a datetime to a date then it was at least two years ago.